Source/Urho3D/Engine/StateManager.cpp

```diff
@@ -41,6 +41,9 @@ namespace Urho3D
 ApplicationState::ApplicationState(Context* context)
     : Object(context)
     , rootElement_(MakeShared<UIElement>(context))
+#if URHO3D_ACTIONS
```

Comment:
Why do we have ActionManager both here and as subsystem?

Reply:
Not every application use states. Imagine you run an action in a game and while it runs you press escape to switch to menu. Menu has its own actions and the action in the game should not update while you are in menu.

Subsystem is for global actions or stateless apps.